### DISCUSSION TITLE
Linux distribution identifiers?

### DISCUSSION THREAD

**Discussion by pmunts** [alire-project/alire#1362] [General]
Where can I find the definitive list of Linux distribution identifiers, such as one would use with `case(distribution)`?

**Comment by mosteo** [comment]
Here: https://github.com/alire-project/alire/blob/ec767ef60d23815df2a1c6954bdd29ce6f6980a0/src/alire/alire-platforms.ads#L36

**Comment by pmunts** [comment]
Is there a way to override platform detection, such as with an environment variable?  I have packages that would work on AlmaLinux and Rocky Linux, but those distributions are not recognized by alr.  It would be useful if there were a way to trick alr into thinking AlmaLinux and Rocky Linux (and CentOS for that matter) were RHEL, which they are supposed to be clones of.

**Reply by mosteo** [reply]
Not currently. You can check the current detection code in file `alire-platforms-current__linux.adb`.

Detection is based on the `id` and `id_like` fields of `/etc/os-release`. In essence, if the `id` is unknown, the `id_like` will be used, which should help in detecting such clones.

We do have an env. var. to force disabling detection, but not to force a particular distro.
